main.tf

```diff
@@ -130,3 +130,21 @@ resource "aws_s3_bucket_inventory" "inventory" {
   optional_fields = ["Size", "LastModifiedDate", "StorageClass", "ETag", "IsMultipartUploaded", "ReplicationStatus", "EncryptionStatus",
   "ObjectLockRetainUntilDate", "ObjectLockMode", "ObjectLockLegalHoldStatus", "IntelligentTieringAccessTier"]
 }
+
+resource "aws_s3_bucket_analytics_configuration" "analytics" {
+  count  = var.analytics_bucket != "" ? 1 : 0
+  bucket = aws_s3_bucket.private_bucket.id
+  name   = "BucketAnalytics"
+
+  storage_class_analysis {
+    data_export {
+      destination {
+        s3_bucket_destination {
+          bucket_arn = var.analytics_bucket
```

Comment:
I was thinking we would stick the analytics in the same bucket this module creates vs a dedicated bucket for storing analytics.

Reply:
I wasn't sure about this -- I thought we might want them in the logging bucket. The example in the Terraform code used two separate buckets, so that's the model I told Misty to use.

Reply:
Chas and I chatted and the aws-logs bucket path makes sense, but we should include a terratest to ensure this actually works. I suspect there will need to be bucket policy changes needed on the aws-logs modules to support his change.

Reply:
Building on a shaky foundation here, but a passing test now exists 🤷‍♀️ . I made a ticket to refactor this test file structure.